client-admin/src/components/conversations-and-account/integrate.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React from "react";
+import { connect } from "react-redux";
+import _ from "lodash";
+import Highlight from "react-highlight";
+import { Box, Heading, Text } from "theme-ui";
+
+@connect((state) => state.user)
+class Integrate extends React.Component {
+  render() {
+    return (
+      <Box>
+        <Box>
+          <Heading
+            as="h3"
+            sx={{
+              fontSize: [3, null, 4],
+              lineHeight: "body",
+              mb: [3, null, 4],
+            }}
+          >
+            Integrate
+          </Heading>
+          <Text>
+            Copy and paste this code into your content management template. Each page (article,
+            post) requires a unique string in the \"PAGE_ID\" field. This should be consistent over
```

Comment:
No need to escape, else backslashes render on page:

```suggestion
            post) requires a unique string in the "PAGE_ID" field. This should be consistent over
```